File: myapp/child/Step20_Lambda.cs
using sfn_ut.sfn.lambda;

namespace sfn_ut.myapp.child;

public class Step20_Lambda : SfnLambda<ChildPayload> {
  public override ChildPayload Execute(ChildPayload input) {
    input.HappyPathData += " > Child 20";
    input.ServiceChargeInDollars = 100;
    return input;
  }
}

File: java-sfn-unit-test/src/main/java/com/sfnut/myapp/child/Step20_Lambda.java
package com.sfnut.myapp.child;

import com.sfnut.sfn.lambda.SfnLambda;

public class Step20_Lambda extends SfnLambda<ChildPayload> {
    @Override
    public ChildPayload execute(ChildPayload input) {
        input.setHappyPathData(input.getHappyPathData() + " > Child 20");
        input.setNestedFromParent("***" + input.getNestedFromParent() + "***");

        return input;
    }
}
